LAKMobile/src/screens/AddJob.tsx

```diff
@@ -176,6 +176,19 @@ export function AddJob({ navigation, route }: AddJobProps) {
     return isAllValid;
   };
 
+  useEffect(() => {
+    getUser(userId, userId).then((user) => {
+      setProfileData(user);
+    });
+  }, [userId]);
+
+  useEffect(() => {
+    setClientName(profileData?.firstName + ' ' + profileData?.lastName);
+    setPhoneNumber(profileData?.phone || '');
+    setPickupLocation(profileData?.location.split(';')[0] || '');
+    setPickupDistrict(profileData?.location.split(';')[1] || PICKER_DEFAULT);
+  }, [profileData]);
+
```

Comment:
Since profileData only changes when userId changes, can we bundle these two useEffect calls into one? Then we wouldn't need an extra state variable since we can just use the result from our API call and set the related state variables inside there

Reply:
Are you saying we should remove the profileData completely, or is something like this okay, where the two useEffects are simply combined? 
![image](https://user-images.githubusercontent.com/78572964/231259771-e72973c3-e07d-4992-a444-e9a01caf507d.png)

Reply:
We followed the example in EditProfileScreen.tsx, where the prepopulate fields is broken into two useEffects
![image](https://user-images.githubusercontent.com/78572964/231260338-02a826f6-338d-410b-a924-cabc0259bc1c.png)

Reply:
Try user.firstName, user.lastName... instead of profileData